services/dynamic-sidecar/tests/unit/test_api_compose.py

```diff
@@ -0,0 +1,178 @@
+# pylint: disable=redefined-outer-name
+# pylint: disable=unused-argument
+
+import json
+from typing import Any, Dict
+
+import pytest
+from async_asgi_testclient import TestClient
+from faker import Faker
+from simcore_service_dynamic_sidecar._meta import api_vtag
+
+DEFAULT_COMMAND_TIMEOUT = 10.0
+
+
+@pytest.fixture
+def compose_spec() -> str:
+    return json.dumps(
+        {
+            "version": "3",
+            "services": {"nginx": {"image": "busybox"}},
+        }
+    )
+
+
+@pytest.mark.asyncio
```

Comment:
FYI: with a single``pytestmark = pytest.mark.asyncio``  in the module, you can avoid all the decorators. SEE [here](https://github.com/ITISFoundation/osparc-simcore/blob/ebae9d04824863dbb3c7f2f7c59fb15acd783c9f/services/api-server/tests/unit/test_api_meta.py#L10)